zenoh/src/net/routing/interceptor/authorization.rs

```diff
@@ -199,10 +199,24 @@ impl PolicyEnforcer {
     ) -> ZResult<PolicyInformation> {
         let mut policy_rules: Vec<PolicyRule> = Vec::new();
         for config_rule in config_rule_set {
+            // config validation
+            if config_rule.interfaces.is_empty()
+                || config_rule.actions.is_empty()
+                || config_rule.flows.is_empty()
+                || config_rule.key_exprs.is_empty()
+            {
+                bail!("error from bad config");
+            }
             for subject in &config_rule.interfaces {
+                if subject.trim().is_empty() {
+                    bail!("error from bad interface value");
```

Comment:
Error message is not specific enough

```suggestion
                    bail!("found an empty interface value in interfaces list");
```

Reply:
Log should start with capital letters.

Reply:
@Mallets this log is concatenated to another one, forming the full log that starts with a capital letter. Same applies for all the logs from this function.